src/elements/LicenseIconography/LicenseIconography.vue

```diff
@@ -1,20 +1,7 @@
 <template>
   <span class="vocab license-icons">
     <template v-for="(icon, index) in processedIconList">
-      <template v-if="showsPopup(index)">
-        <Popup
-          :key="index"
-          v-bind="$attrs">
-          <FontAwesomeIcon
-            :icon="['fab', icon]"
-            fixed-width/>
-          <template #popup>
-            <!-- @slot Popup content goes here -->
-            <slot :name="index">{{ computedStrings[index] }}</slot>
-          </template>
-        </Popup>
-      </template>
-      <template v-else>
+      <template>
         <FontAwesomeIcon
```

Comment:
The `v-for` from the parent `<template>` tag can be moved here, making that one redundant as well.

Reply:
done